### PR TITLE
Changed font urls to relative, removed global.css

### PR DESCRIPTION
**Pull request by grahhham**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -92,11 +92,11 @@ const Home = ({ stars }: Props) => {
...
 
 Home.getInitialProps = async () => {
-	const res = await fetch("https://api.github.com/repos/zeit/next.js");
-	const json = await res.json();
+	// const res = await fetch("https://api.github.com/repos/zeit/next.js");
+	// const json = await res.json();
 	return {
 		namespacesRequired: includeDefaultNamespaces(["common"]),
-		stars: json.stargazers_count,
+		// stars: json.stargazers_count,
```
Comment (nedga055) [inline pages/index.tsx:99]: Having this removed will break the template that uses the `stars` attribute. Either leave the API call, or remove the attribute from the component. I'd be inclined to leave the API call for reference purposes.

**Review by nedga055 — CHANGES_REQUESTED** [review]
Just one change.

File: pages/index.tsx (hunk 1 above)
Comment (grahhham) [reply pages/index.tsx]: Okay, re-added! Commented out just because it crashes program on prem

**Review by nedga055 — APPROVED** [review]

**Comment by nedga055** [comment]
Good to go!

**Pull request merged**